packages/app/src/ui/components/VerifiableCredential/Item/VerifiableCredentialItem.tsx

```diff
@@ -18,25 +18,25 @@ export interface VerifiableCredentialItemProps {
   verifiableCredential: IVerifiableCredential;
   metadata: IVerifiableCredentialMetadata;
   selected?: boolean;
-  onRenameVerifiableCredential?: (hash: string, name: string) => Promise<void>;
-  onDeleteVerifiableCredential?: (hash: string) => Promise<void>;
-  onToggleSelectVerifiableCredential?: (hash: string) => void;
+  onRenameVC?: (hash: string, name: string) => Promise<void>;
+  onDeleteVC?: (hash: string) => Promise<void>;
+  onToggleSelectVC?: (hash: string) => void;
```

Comment:
```suggestion
  onRename?: (hash: string, name: string) => Promise<void>;
  onDelete?: (hash: string) => Promise<void>;
  onToggleSelect?: (hash: string) => void;
```

Reply:
I updated all instances of callbacks passed from hooks to remove "VC". I'm following the pattern of appending "VC" to functions passed in as arguments into the hook to avoid duplicate identifier.